cookbooks/aws-parallelcluster-environment/resources/nfs/nfs_alinux2.rb

```diff
@@ -29,7 +29,7 @@
 action_class do
   def override_server_template
     edit_resource(:template, node['nfs']['config']['server_template']) do
-      source 'nfs/nfs.conf.erb'
+      source 'nfs/nfs.erb'
```

Comment:
can we use two different names like: `nfs/nfs.ubuntu.erb` and `nfs/nfs.ubuntu22+.erb` to be able to distinguish them in a easy way

Reply:
sure, the `nfs.erb` is used for all the operating systems, I can create some more reader friendly names